src/frontend/komponenter/Journalforing/EndreDokumentTittel.tsx

```diff
@@ -0,0 +1,86 @@
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import CreatableSelect from 'react-select/creatable';
+import { dokumentTitler } from './konstanter/dokumenttitler';
+
+const StyledKnapper = styled.div`
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 0.5rem;
+`;
+
+const StyledHovedKnapp = styled(Hovedknapp)`
+    margin-right: 0.25rem;
+`;
+
+const StyledDiv = styled.div`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+const StyledCreatableSelect = styled(CreatableSelect)``;
+
+const customStyles = {
+    container: (styles: Record<string, string>, state: Record<string, string>) => ({
+        ...styles,
+        backgroundColor: '#fff',
+        borderRadius: 4,
+        boxShadow: state.isFocused ? '0 0 0 3px #254b6d' : '',
+        border: '1px solid #78706a',
+    }),
+    control: (styles: Record<string, string>) => ({
```

Comment:
control er ikke brukt

Reply:
custom styles sends in som en prop til CreatableSelect fordi det er måten att override defaultstylingen til den komponenten... så den ær i bruk